### PR TITLE
Stub state diagram generation

### PR DESCRIPTION
**Pull request by despo**
Spec takes ~10 seconds to run and not testing the generated diagram
Added expectations to ensure that calls to service still take place.

### CODE REVIEW

File: spec/system/support_interface/docs_spec.rb [hunk 1]
```diff
@@ -18,7 +18,10 @@ def given_i_am_a_support_user
   end
 
   def when_i_visit_the_process_documentation
+    allow(ApplicationStateChange).to receive(:workflow_spec).and_return(Struct.new(:states).new([]))
+
     visit support_interface_docs_provider_flow_path
+    expect(ApplicationStateChange).to have_received(:workflow_spec).exactly(3).times
```
Comment (cwrw) [inline spec/system/support_interface/docs_spec.rb:24]: Maybe we can move this to its own `then` step, as I think usually we should avoid any expectations in `when` steps, what do you think?

**Review by cwrw — COMMENTED** [review]
This is great, spec went from 10 sec to 0.2 sec, just a comment around the stpes

File: spec/system/support_interface/docs_spec.rb (hunk 1 above)
Comment (despo) [reply spec/system/support_interface/docs_spec.rb:29]: Good point. I have moved the expectations to their own `then` steps.

**Review by cwrw — APPROVED** [review]

**Review by tijmenb — COMMENTED** [review]
I wonder if we should stub the SVG generation at a higher level, would stubbing `allow(StateDiagram).to receive(:svg)` work as well? That avoids stubbing the internals.

**Comment by tijmenb** [comment]
Hmm, on second thought, this change removes all content from the flow pages, so the code is excercised at all. Is that the intention? In that case, we should remove all tests except the emails, because they aren't doing anything.

**Pull request merged**